substrate/frame/nomination-pools/src/lib.rs

```diff
@@ -3302,6 +3381,36 @@ impl<T: Config> Pallet<T> {
 		Ok(())
 	}
 
+	/// Slash member against the pending slash for the pool.
+	fn do_apply_slash(
+		member_account: &T::AccountId,
+		reporter: Option<T::AccountId>,
+	) -> DispatchResult {
+		// calculate points to be slashed.
+		let member =
+			PoolMembers::<T>::get(&member_account).ok_or(Error::<T>::PoolMemberNotFound)?;
+
+		let pool_account = Pallet::<T>::create_bonded_account(member.pool_id);
```

Comment:
A better name for this fn would be `generate_bonded_account_id` or something like that. Create account sounds like funding it or something similar.

Reply:
Yeah it confuses me too.

Reply:
Yeah I agree with this, it's pretty `create` confusing here.